src/app/feedback/feedback.service.ts

```diff
@@ -0,0 +1,38 @@
+import { ActivatedRoute, Router, UrlSegment } from '@angular/router';
+import { AngularFireDatabase, FirebaseListObservable } from 'angularfire2/database';
+import { Injectable } from '@angular/core';
+import { Message } from './message';
+import { Observable } from 'rxjs/Observable';
+import { of } from 'rxjs/Observable/of';
+
+@Injectable()
+export class FeedbackService {
+  private repo$: FirebaseListObservable<any>;
```

Comment:
Can you add a type? If we don't have one yet, then don't worry about it. I don't think we do...